core/app/components/forms/elements/RelatedPubsElement.tsx

```diff
@@ -94,8 +94,11 @@ export const ConfigureRelatedValue = ({
 	onBlur?: () => void;
 	className?: string;
 }) => {
-	const configLabel = "label" in element.config ? element.config.label : undefined;
-	const label = configLabel || element.label || slug;
```

Comment:
`slug` is not actually a field slug, it's the `name` for an `<input>` managed by react hook form, so it's not really appropriate for displaying to users even as a fallback